### PR TITLE
test: In Team Event type when location is Organizer Default App and U…

### LINKED ISSUE #11776: [CAL-2598] Write test for meeting location when user metadata is empty
Test for this bug :- https://threads.com/34541021924?s=mEHjgC1CnyVWpDJifMcL7C

Hotifix:- https://github.com/calcom/cal.com/pull/11735

<sub>[CAL-2598](https://linear.app/calcom/issue/CAL-2598/write-test-for-meeting-location-when-user-metadata-is-empty)</sub>

### PR DESCRIPTION
**Pull request by Udit-takkar**
Fixes: https://github.com/calcom/cal.com/issues/11776

How to test ?

yarn test collective-scheduling.test.ts

### CODE REVIEW

File: packages/features/bookings/lib/handleNewBooking.ts [hunk 1]
```diff
@@ -1172,7 +1172,7 @@ async function handler(
       organizerOrFirstDynamicGroupMemberDefaultLocationUrl =
         organizerMetadata?.defaultConferencingApp?.appLink;
     } else {
-      locationBodyString = "";
+      locationBodyString = "integrations:daily";
```
Comment (Udit-takkar) [inline packages/features/bookings/lib/handleNewBooking.ts:1175]: Default is Daily video if no default conferencing app

File: packages/features/bookings/lib/handleNewBooking/test/team-bookings/collective-scheduling.test.ts [hunk 2]
```diff
@@ -1087,6 +1090,212 @@ describe("handleNewBooking", () => {
         timeout
       );
 
+      test(
+        `In Team Event type when location is Organizer Default App and User Metadata is {} (Empty Object).`,
```
Comment (joeauyeung) [inline packages/features/bookings/lib/handleNewBooking/test/team-bookings/collective-scheduling.test.ts:1094]: NIT: I think when looking at the title of the test it should be very descriptive what's happening. Could we change this to something like "When event type location is Organizer Default App and user metadata is empty, default to Cal Video".

File: packages/features/bookings/lib/handleNewBooking/test/team-bookings/collective-scheduling.test.ts [hunk 3]
```diff
@@ -1087,6 +1090,212 @@ describe("handleNewBooking", () => {
...
+          expectBookingCreatedWebhookToHaveBeenFired({
+            booker,
+            organizer,
+            location: OrganizerDefaultConferencingAppType,
+            subscriberUrl: "http://my-webhook.example.com",
+            videoCallUrl: `${WEBAPP_URL}/video/${createdBooking.uid}`,
+          });
+        },
+        timeout
+      );
```
Comment (joeauyeung) [inline packages/features/bookings/lib/handleNewBooking/test/team-bookings/collective-scheduling.test.ts:1297]: Wasn't sure if these are necessary but as long as we're checking the right location URL was passed I'm ok with it.

**Review by joeauyeung — DISMISSED** [review]
I'm ok to approve, tests are passing. I did have some NITs though.

**Review by CarinaWolli — APPROVED** [review]
Looks good, new test is passing 👍

**Pull request merged**